tests/lax_numpy_test.py

```diff
@@ -1351,6 +1351,54 @@ def testPad(self, shape, dtype, mode, pad_width, constant_values, rng_factory):
                             check_dtypes=shape is not jtu.PYTHON_SCALAR_SHAPE)
     self._CompileAndCheck(jnp_fun, args_maker)
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_shape={}_mode={}_pad_width={}_stat_length={}".format(
+          jtu.format_shape_dtype_string(shape, dtype), mode, pad_width, stat_length),
+       "shape": shape, "dtype": dtype, "mode": mode, "pad_width": pad_width,
+       "stat_length": stat_length, "rng_factory": jtu.rand_default}
+      for mode, shapes in [
+          ('maximum', nonempty_shapes),
+          ('minimum', nonempty_shapes),
+          ('mean', nonempty_shapes),
+          ('median', nonempty_shapes),
+      ]
+      for shape, dtype in _shape_and_dtypes(shapes, all_dtypes)
```

Comment:
since all shapes here are the same, you could simplify this and pass `nonempty_shapes` directly here.